### PR TITLE
Feat back tao

### PR DESCRIPTION
**Pull request by SpliiT**

### CODE REVIEW

File: src/geodeapp_back/config.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    CORS_HEADERS = "Content-Type"
+    UPLOAD_FOLDER = "./uploads"
+    WORKFLOWS_DATA_FOLDER = "./data_workflows/"
+    LOCK_FOLDER = "./lock/"
+    TIME_FOLDER = "./time/"
+
+
+class ProdConfig(Config):
+    SSL = None
+    ORIGINS = ["https://geode-solutions.com", "https://next.geode-solutions.com"]
```
[the remark this answers is not in the document]
Comment (SpliiT) [reply src/geodeapp_back/config.py:19]: sure

File: src/geodeapp_back/config.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    MINUTES_BEFORE_TIMEOUT = "5"
+    SECONDS_BETWEEN_SHUTDOWNS = "150"
+    DATA_FOLDER = "/data/"
+
+
+class DevConfig(Config):
+    SSL = None
+    ORIGINS = "http://localhost:3000"
+    MINUTES_BEFORE_TIMEOUT = "1000"
```
[the remark this answers is not in the document]
Comment (SpliiT) [reply src/geodeapp_back/config.py:28]: we'll see

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,10 @@
+__pycache__
+.env
+env
+venv
+tests
+uploads
+.vscode
+lock
+Architecture.drawio
```
[the remark this answers is not in the document]
Comment (SpliiT) [reply .dockerignore:9]: it's good

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "geodeapp_back"
+version = "0.0.1"
```
Comment (BotellaA) [inline pyproject.toml:8]: is it updated with semantic release?

File: src/geodeapp_back/app.py [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+if os.path.isfile("./.env"):
+    basedir = os.path.abspath(os.path.dirname(__file__))
+    dotenv.load_dotenv(os.path.join(basedir, ".env"))
+
+""" Global config """
+app = flask.Flask(__name__)
+
+
+def kill_task():
```
Comment (BotellaA) [inline src/geodeapp_back/app.py:29]: this function and the exception handler could be in OGW-Back?

**Review by BotellaA — APPROVED** [review]

File: src/geodeapp_back/app.py (hunk 5 above)
Comment (SpliiT) [reply src/geodeapp_back/app.py:29]: yes

**Pull request merged**